stytch/core/http/client.py

```diff
@@ -66,6 +68,17 @@ def post(
         resp = requests.post(url, json=json, headers=final_headers, auth=self.auth)
         return self._response_from_request(resp)
 
+    def postForm(
```

Comment:
can we name this `post_form` instead? We use snake case everywhere else in this package